.github/workflows/build_all.yml

```diff
@@ -158,11 +161,15 @@ jobs:
       # github actions overwrites brew's python. Force it to reassert itself, by running in a separate step.
       - name: unbreak python in github actions
         run: |
-          find /usr/local/bin -lname '*/Library/Frameworks/Python.framework/*' -delete
+          case "${{ matrix.platform }}" in
+          arm64) homebrew=/opt/homebrew ;;
+          x86_64) homebrew=/usr/local ;;
+          esac
+          find $homebrew/bin -lname '*/Library/Frameworks/Python.framework/*' -delete
           sudo rm -rf /Library/Frameworks/Python.framework/
           brew install --force python3 && brew unlink python3 && brew unlink python3 && brew link --overwrite python3
           # Work around PEP 668 nonsense…
-          find /usr/local/Cellar/python* -name EXTERNALLY-MANAGED -print0 | xargs -0 rm -vf
+          find $homebrew/Cellar/python* -name EXTERNALLY-MANAGED -print0 | xargs -0 rm -vf
```

Comment:
Can't we get the install location from homebrew (`brew --prefix` maybe?)

Reply:
Probably. In meson.git's CI we add `$(brew --prefix llvm)/bin` to PATH, for example.

Reply:
Ah yes, good call.  Fixed.